mlir/test/Ion/QuantumToIon.mlir

```diff
@@ -14,6 +14,9 @@
 
 // RUN: quantum-opt %s --quantum-to-ion --split-input-file -verify-diagnostics | FileCheck %s
 
+// COM: the physical parameters come from
+// COM: frontend/catalyst/third_party/oqd/src/oqd_gate_decomposition_parameters.toml
```

Comment:
@joeycarter I leave it to you to add tests for gates on other wires?

Reply:
Yes, working on this now!